lib/Conch/Controller/Schema.pm

```diff
@@ -0,0 +1,60 @@
+package Conch::Controller::Schema;
+
+use Mojo::Base 'Mojolicious::Controller', -signatures;
+
+use Role::Tiny::With;
+with 'Conch::Role::MojoLog';
+
+use Mojo::Exception;
+use Data::Visitor::Tiny qw(visit);
+use Conch::Plugin::JsonValidator;
+
+=pod
+
+=head1 NAME
+
+Conch::Controller::User
+
+=head1 METHODS
+
+=head2 get
+
+Get the json-schema in JSON format.
+
+=cut
+
+sub get ($c) {
+	my $type = $c->param('input_or_response');
+	my $name = $c->param('name');
```

Comment:
Values pulled out of the path itself are better taken as `$c->stash($name)`, with param reserved for form parameters.

Reply:
Fixed in fdeef31